lectures/tools_and_techniques/stationary_densities.md

```diff
@@ -978,18 +979,18 @@ Random.seed!(42);  # reproducible results
 ```{code-cell} julia
 s = 0.2
 δ = 0.1
-a_σ = 0.4  # A = exp(B) where B ~ N(0, a_σ)
-α = 0.4    # We set f(k) = k**α
+a_sigma = 0.4  # A = exp(B) where B ~ N(0, a_sigma)
+alpha = 0.4    # We set f(k) = k**alpha
 ψ_0 = Beta(5.0, 5.0)  # Initial distribution
-ϕ = LogNormal(0.0, a_σ)
+ϕ = LogNormal(0.0, a_sigma)
 
 function p_growth(x, y)
     # Stochastic kernel for the growth model with Cobb-Douglas production.
     # Both x and y must be strictly positive.
 
-        d = s * x.^α
+    d = s * x .^ alpha
 
-    pdf_arg = clamp.((y .- (1-δ) .* x) ./ d, eps(), Inf)
+    pdf_arg = clamp.((y .- (1 - δ) .* x) ./ d, eps(), Inf)
```

Comment:
yeah, `δ => "delta"`

Reply:
I also saw somethign in this file with `θ = 0.8`  so we should add in `θ => "theta"` as well.